opentelemetry-otlp/src/exporter/http/mod.rs

```diff
@@ -291,6 +312,26 @@ fn resolve_endpoint(
         .map_err(From::from)
 }
 
+fn add_header_from_string(input: &str, headers: &mut HashMap<HeaderName, HeaderValue>) {
+    for pair in input.split(',') {
```

Comment:
nit: you can use [split_terminator](https://doc.rust-lang.org/stable/std/primitive.str.html#method.split_terminator) here to avoid the empty last instance check

Reply:
Thanks done.